troubleshooting.adoc

```diff
@@ -72,12 +72,12 @@ system:
 sudo apt-get install build-essential uuid-dev zlib1g-dev python-dev
 --------------------------------------------------------------------
 
-Download protobuf version 2.5.0 or later from link:http://code.google.com/p/protobuf/[protobuf]
+Download protobuf version 3.0.0 or later from link:http://code.google.com/p/protobuf/[protobuf]
```

Comment:
Hi @destijl  

When I installed the compiler using sudo apt install protobuf-compiler, (Ubuntu 16.04), it installed v2.5.0. In the next step, cd /grr,  pip install -editable . it gave an error "v3.0.0 required", so I submitted the PR. I will verify it once again.

Thanks